src/org/ods/OdsPipeline.groovy

```diff
@@ -205,7 +205,7 @@ class OdsPipeline implements Serializable {
             --request POST \\
             --header \"Content-Type: application/json\" \\
             --data '{\"state\":\"${state}\",\"key\":\"${buildName}\",\"name\":\"${buildName}\",\"url\":\"${context.buildUrl}\"}' \\
-            https://${context.bitbucketHost}/rest/build-status/1.0/commits/${context.gitCommit}
+            ${context.bitbucketUrl}/rest/build-status/1.0/commits/${context.gitCommit}
```

Comment:
Can we have a variable to set a --insecure flag? That could help if for any reason the repo is behind a self-signed certificate

Reply:
Different issue, different PR :)